websites/S/SlimeRead/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "1190448612778463242",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/Ht1LSYT.png",
```

Comment:
This is without the background:
```suggestion
			largeImageKey: "https://i.imgur.com/x6NuCdk.png",
```